internal/stack/certs.go

```diff
@@ -29,6 +29,7 @@ var tlsServices = []tlsService{
 	{Name: "package-registry"},
 	{Name: "fleet-server"},
 	{Name: "logstash"},
```

Comment:
Nit. Maybe we can add another option to generate the PKCS8 files here, so we don't need to generate them later on container startup.
```suggestion
	{Name: "logstash", GeneratePKCS8: true},
```

Reply:
Can try this in a new PR